src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryText.cs

```diff
@@ -1059,6 +1071,34 @@ public static FormulaValue OptionSetValueToLogicalName(IRContext irContext, Opti
             var optionSet = args[0];
             var logicalName = optionSet.Option;
             return new StringValue(irContext, logicalName);
+        }
+
+        public static FormulaValue PlainText(IRContext irContext, StringValue[] args)
+        {        
+            //Decode html specific characters
+            string text = WebUtility.HtmlDecode(args[0].Value.Trim());

```

Comment:
>string text = WebUtility.HtmlDecode(args[0].Value.Trim()); [](http://example.com/codeflow?start=12&length=58)

This may cause double decoding. For example, `PlainText("Hello&lt;br/&gt;world")` should return `"Hello<br/>world"` but it now adds a newline where it shouldn't.

Reply:
Move this to the end of function to not double decoding.